api/Performance.json

```diff
@@ -195,9 +195,16 @@
                 "version_removed": "2.0"
               }
             ],
-            "webview_android": {
-              "version_added": true
-            }
+            "webview_android": [
+              {
+                "version_added": "≤37"
+              },
+              {
+                "version_added": "≤37",
```

Comment:
I'm not too sure about adding ranges like this which give very little information about when it was supported.